docs/setting_up/exporting.md

```diff
@@ -1,6 +1,6 @@
-# Exporting Data from an ActivePivot Application
+# Exporting Data from an Atoti Server Application
 
-The exporting of an ActivePivot application is done through an
+The exporting of an Atoti Server application is done through an
```

Comment:
```suggestion
The exporting of an Atoti application is done through an
```
You got the idea. I think that with this one, I made a comment for each one to change.